README.md

```diff
@@ -51,3 +51,19 @@ use the option `-b`. This can be used for running the e2e tests manually.
 
 ## Running tests with delegation fees
 If you want to run tests with delegation fees provided the `-f` flag.
+
+## Running tests against specific (relay, clientlib, index, contracts) versions
+Per default the e2e tests pull the latest version for all used images from docker hub.
```

Comment:
That's not true, it only pulls if you do -p, otherwise it will use the local image tagged `trustlines/relay:latest` ...

Reply:
ah okay, I'll correct that.